LabController/src/bkr/labcontroller/log_storage.py

```diff
@@ -1,37 +1,38 @@
-
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 
 # We are talking about job logs here, not logs produced by the daemons.
 
-import os, os.path
 import errno
+import os
+import os.path
+
 from bkr.common.helpers import makedirs_ignore
 
-class LogFile(object):
 
+class LogFile(object):
     def __init__(self, path, register_func, create=True):
-        self.path = path #: absolute path where the log will be stored
-        self.register_func = register_func #: called only if the file was created
-        self.create = create #: create the file if it doesn't exist
+        self.path = path  #: absolute path where the log will be stored
+        self.register_func = register_func  #: called only if the file was created
+        self.create = create  #: create the file if it doesn't exist
 
     def __repr__(self):
-        return '%s(%r)' % (self.__class__.__name__, self.path)
+        return "%s(%r)" % (self.__class__.__name__, self.path)
 
     def open_ro(self):
         """
         If you just want to read the log, call this instead of entering the context manager.
         """
-        return open(self.path, 'r')
+        return open(self.path, "r")
 
     def __enter__(self):
         makedirs_ignore(os.path.dirname(self.path), 0o755)
         created = False
         if self.create:
             try:
-                # stdio does not have any mode string which corresponds to this 
+                # stdio does not have any mode string which corresponds to this
```

Comment:
You must be running a script to produce these changes.  They're too consistent to be humanly done.